src/ProcessManagerBundle/Factory/ProcessFactory.php

```diff
@@ -45,8 +45,14 @@ public function createProcess(
         string $type = null,
         string $message = '',
         int $total = 1,
-        int $progress = 0
+        int $progress = 0,
+        int $started = -1,
+        int $completed = 0
```

Comment:
can we add a trailing comma here

Reply:
I am not sure I understand. You want a comma after $completed = 0? What is the purpose?

I did notice another error related to this method. I forgot to add the new arguments in ProductFactoryInterface

Reply:
forget about that, thats a PHP7.3 feature that allows trailing commas in functions